packages/node_modules/@node-red/editor-client/src/js/ui/palette-editor.js

```diff
@@ -1462,6 +1473,63 @@ RED.palette.editor = (function() {
         })
     }
 
+    const updateStatusWidget = $('<button type="button" class="red-ui-footer-button red-ui-update-status"></button>');
+    let updateAvailable = [];
+
+    function addUpdateInfoToStatusBar() {
+        updateStatusWidget.on("click", function (evt) {
+            RED.actions.invoke("core:manage-palette", {
+                view: "nodes",
+                filter: '"' + updateAvailable.join('", "') + '"'
```

Comment:
I appreciate we now have lots of PRs open in this area and there are overlapping things. For example, the ability to pass in this filter is dependent on the PR to manage import/export of module lists.

We also have the 'update all' PR - which I haven't looked at yet.

I suspect my feedback on that will include a way to one-click filter the view to only show modules that need updating - and once we have that, then this line can be updated to set a flag to filter by 'has:update' or something like that, rather than pass in a list of module names.

So, not asking for any changes from you here - just making an observation this bit may change as we work through the other PRs.